### PR TITLE
estimate gas correctly for blob tx in increaseGasPrice

### PR DESCRIPTION
**Pull request by zhiqiangxu**
This PR makes estimating gas work correctly for blob tx in `increaseGasPrice`.

It's similar to how it's done in `craftTx` [here](https://github.com/ethereum-optimism/optimism/blob/72eb3116c2f6ff573099a8cafc8f35415138850c/op-service/txmgr/txmgr.go#L377-L381).

(This PR can also be considered a left-over of [this](https://github.com/ethereum-optimism/optimism/pull/12086) one.)

### CODE REVIEW

File: op-service/txmgr/txmgr.go [hunk 1]
```diff
@@ -797,14 +797,30 @@ func (m *SimpleTxManager) increaseGasPrice(ctx context.Context, tx *types.Transa
...
+	callMsg := ethereum.CallMsg{
 		From:      m.cfg.From,
 		To:        tx.To(),
 		GasTipCap: bumpedTip,
 		GasFeeCap: bumpedFee,
 		Data:      tx.Data(),
 		Value:     tx.Value(),
-	})
+	}
+	var bumpedBlobFee *big.Int
```
Comment (geoknee) [inline op-service/txmgr/txmgr.go:814]: Nit: could this var be moved inside the scope of the following `if` block? That is, if we don't need it up at this level of scope.

Comment (zhiqiangxu) [reply op-service/txmgr/txmgr.go:814]: It is used [here](https://github.com/zhiqiangxu/optimism/blob/5df2ee636c550e8942dd56553fc9255830ae5a65/op-service/txmgr/txmgr.go#L857), so can't be moved inside `if`.

**Comment by zhiqiangxu** [comment]
@geoknee Any further advice for this PR?

**Review by geoknee — APPROVED** [review]

**Comment by mslipper** [comment]
/ci authorize https://github.com/ethereum-optimism/optimism/pull/13116/commits/5df2ee636c550e8942dd56553fc9255830ae5a65

**Comment by mslipper** [comment]
@zhiqiangxu please rebase, a lot of the test errors here should be fixed.

**Comment by zhiqiangxu** [comment]
> @zhiqiangxu please rebase, a lot of the test errors here should be fixed.

Done.

**Comment by mslipper** [comment]
/ci authorize https://github.com/ethereum-optimism/optimism/pull/13116/commits/a8bf3887feb11a7e7d1ec7a92d4cc76e8ed07d81

**Pull request merged**